src/AutoRest.CSharp/Output/Models/Requests/LongRunningOperation.cs

```diff
@@ -48,20 +47,20 @@ public LongRunningOperation(OperationGroup operationGroup, Operation operation,
                 {
                     NextPageMethod = Client.RestClient.GetNextOperationMethod(operation.Requests.Single());
                     PagingResponse = new PagingResponseInfo(paging, ResultType);
-                    ResultType = new CSharpType(typeof(AsyncPageable<>), PagingResponse.ItemType);
+                    ResultType = new CSharpType(typeof(Azure.AsyncPageable<>), PagingResponse.ItemType);
                 }
             }
             else
             {
-                ResultType = typeof(Response);
+                ResultType = typeof(Azure.Response);
```

Comment:
Was there a type conflict?

Reply:
Yes, with latest Azure.Core. Want me to fight it harder or just accept?

Reply:
Can you try adding back the using and see what's the type that conflicts?

Reply:
```
LongRunningOperation.cs(19,68): error CS0104: 'Operation' is an ambiguous reference between 'AutoRest.CSharp.Input.Operation' and 'Azure.Operation
```

I reduced this down to one more specific type reference.